ClosedXML_Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -192,6 +192,23 @@ public void InsertData_with_Nulls_IEnumerable()
             Assert.AreEqual(String.Empty, ws.Cell("A5").Value);
         }
 
+        [Test]
+        public void InsertData_Objects_with_KnownTypeMapping()
```

Comment:
Besides `enum`s that can be either `int` or `string`, are there other use cases of this enhancement?

Reply:
As mentioned we do have a type called `Date`. But this is a specific struct.

With version 0.95.3 this type got exported as a string. We can't export the data with the newest version. There is always the exception "Unable to set cell value to...". With this small extension we could export it as `DateTime`.

Reply:
I can make a test with `Date` as well. I'm just not sure if it is worth it to include this type.

Reply:
Yes, please do the `Date` struct as well so that I can understand. @Pankraty has some alternative ideas on how to handle this.

Reply:
Sorry, with Version 0.92.1 the unknown types got exported as string. 0.95.3 is the newest version.